proto/expr/BUILD.bazel

```diff
@@ -0,0 +1,61 @@
+package(default_visibility = ["//visibility:public"])
+
+##############################################################################
+# Common
+##############################################################################
+
+proto_library(
+    name = "expr_proto",
+    srcs = [
+        "checked.proto",
+        "eval.proto",
+        "explain.proto",
+        "syntax.proto",
+        "value.proto",
+    ],
+    deps = [
+        "@com_google_googleapis//google/rpc:status_proto",
```

Comment:
It might be a good idea to split the `checked.proto` and `syntax.proto` from the other protos so that upstream dependencies don't have to pick up a dep on googleapis.

Reply:
Done

Reply:
@TristonianJones PTAL when you get a chance